.claude/hooks/post_tool_use_commit_only.py

```
@@ -0,0 +1,115 @@
+
+    # Find project root
+    project_root = find_project_root() or Path.cwd()
+
+    # Get all modified files
+    modified_files = get_all_modified_files(
+        tool_name, tool_input, tool_response, project_root
+    )
+
+    # Auto-commit is enabled by default for this simplified hook
+    # but can be disabled with environment variable
+    if get_bool_env("DISABLE_AUTO_COMMIT", False):
```

Ruff (0.12.2)

Boolean positional value in function call

(FBT003)